src/retinanet.py

```diff
@@ -228,20 +236,32 @@ def forward(self, output, target):
         masked_loc_targets = loc_targets[mask].view(-1,4)  # [#pos,4]
 
         pos_neg = cls_targets > -1  # exclude ignored anchors
+        num_pos_neg = pos_neg.data.long().sum()
```

Comment:
@taraspiotr are those improvements from @apyskir PR ?

Reply:
@jakubczakon no, I introduced `num_pos_neg` to weight final loss as cls_loss / num_pos_neg + loc_loss / num_pos